robot/util/emailer/emailIPAddress.js

```diff
@@ -9,7 +9,8 @@ const EMAILS_FILE_NAME = 'emails.txt'
 // fallback to default emails if there's and issue with the emails file
 const DEFAULT_EMAILS = [
     'davidhuculak5@gmail.com',
-    'petergranitski@gmail.com'
+    'petergranitski@gmail.com',
+    'william.wells@spaceconcordia.ca'
```

Comment:
lol pls add me

Reply:
Just so you know, read this code to see how to place the email config file so that more members can be notified

Reply:
Done